ffi/diplomat/src/bin/completeness.rs

```diff
@@ -91,6 +108,45 @@ lazy_static::lazy_static! {
         "Yokeable",
         "ZeroFrom",
     ].into_iter().collect();
+
+    static ref ALLOWLISTED_PATHS: HashSet<Vec<String>> = [
```

Comment:
I think "allow" is not a good phrase (are we "allowing" it to be excluded). I'd call it IGNORELIST